projects/packages/forms/src/dashboard/components/InboxStatusToggle/useStatusTabs.js

```diff
@@ -0,0 +1,43 @@
+import { useEntityRecords } from '@wordpress/core-data';
+import { __, sprintf } from '@wordpress/i18n';
+
+/**
+ * Returns a formatted tab label with count.
+ *
+ * @param {string} label - The label for the tab.
+ * @param {number} count - The count to display.
+ * @return {string} The formatted label.
+ */
+function getTabLabel( label, count ) {
+	/* translators: 1: Tab label, 2: Count */
+	return sprintf( __( '%1$s (%2$s)', 'jetpack-forms' ), label, count || 0 );
+}
+
+/**
+ * Custom hook to get status tabs for the inbox view.
+ *
+ * @return {Array<{label: string, value: string}>} The status tabs.
+ */
+export default function useStatusTabs() {
+	const { totalItems: totalItemsInbox } = useEntityRecords( 'postType', 'feedback', {
+		status: 'publish,draft',
+		per_page: 1,
+		_fields: 'id',
+	} );
+	const { totalItems: totalItemsSpam } = useEntityRecords( 'postType', 'feedback', {
+		status: 'spam',
+		per_page: 1,
+		_fields: 'id',
+	} );
+	const { totalItems: totalItemsTrash } = useEntityRecords( 'postType', 'feedback', {
```

Comment:
Previous version pulled `isResolving` from `useEntityRecords` to show loading state. Should we handle slow loading here too somehow visually?

Likely fine to show tabs at all times, but the number could be [pulsating](https://wordpress.github.io/gutenberg/?path=/docs/components-animate--docs#loading) grey box?

Not a blocker for merging and can be follow-up.